CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.5.1
+* Upgrade podfile to support latest Flutter version
```

Comment:
Keep an empty line after a headline.

Consider spelling "Podfile" as it's actually called (but it's clear what you're talking about anyway).

I'd use the word "Update" rather then "Upgrade" here, but I'm not a native speaker :D (upgrade hardware, upgrade software, but update configuration—I guess 🤷‍♂️)

Reply:
Yuk the empty line was a blocker for me also fixed the other minors